example02/index.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<head>
+    <meta charset="utf8">
+    <title>innerself example 02</title>
+    <style>
+        * {margin: 0; padding: 0; box-sizing: border-box}
+
+        html, body, #app, #app > div {width: 100%; height: 100%}
+
+        body > h1 {position: absolute; top: 1rem; left: 0; width: 100%; text-align: center}
+
+        body > h1 > span {font-size: .6em}
+
+
+        #app > button{
+            position: fixed;
+            bottom: 1rem;
+            left: 50%;
+            margin-left: -5rem;
+            width: 10rem;
+        }
+
+        #app .floating-box{
+            position: absolute;
+            width: 5rem;
+            height: 5rem;
+            cursor: pointer;
+        }
+
+        #app .floating-box > span{
```

Comment:
You won't need th `span` not this CSS if you add the following to `.floating-box`:

```css
.floating-box {
    /* … other styles here … */
    display: flex;
    justify-content: center;
    align-items: center;
    text-align: center;
}
```